src/explore-education-statistics-admin/src/components/ProtectedRoute.tsx

```diff
@@ -47,43 +54,12 @@ const AuthenticationCheckingComponent = ({
     );
   }
 
-  if (typeof protectedByAction !== 'undefined' && protectedByAction) {
+  if (accessDenied) {
     handleManualErrors.forbidden();
     return null;
   }
 
-  if (typeof protectedByAction !== 'undefined' && !protectedByAction) {
-    return React.createElement(component, props);
-  }
-
-  return null;
-};
-
-/**
- * Creates a <Route> that firstly checks the user's authentication
- * status and then renders the protected component if the user has been
- * successfully authenticated, or redirects the user to the sign-in page
- * if in need of authentication.
- */
-const ProtectedRoute = ({
-  component,
-  allowAnonymousUsers = false,
-  protectionAction,
-  ...rest
-}: ProtectedRouteProps) => {
-  return (
-    <Route
-      {...rest}
-      render={(props: RouteComponentProps) => (
-        <AuthenticationCheckingComponent
-          {...props}
-          component={component}
-          allowAnonymousUsers={allowAnonymousUsers}
-          protectionAction={protectionAction}
-        />
-      )}
-    />
-  );
+  return <Route component={component} {...rest} />;
```

Comment:
Would get rid of the check on `component` and then we can just do `<Route {...rest} />`